runbooks/pipeline/sentry-triage.md

```diff
@@ -4,11 +4,17 @@
 > * [Fingerprinting and grouping](https://docs.sentry.io/product/sentry-basics/grouping-and-fingerprints/)
 > * [Merging issues](https://docs.sentry.io/product/data-management-settings/event-grouping/merging-issues/)
 
-Once a day, we should check Sentry issues created since the prior day, using the following query.
+Once a day, the person responsible for triage should check Sentry for new and current issues. There are two separate things to check:
 
-`is:unresolved firstSeen:-24h ` ([saved search link](https://sentry.calitp.org/organizations/sentry/issues/searches/5/?environment=cal-itp-data-infra&project=2))
+* All **new issues** from the past 24 hours. An issue is a top-level error/failure/warning, and a new issue represents something we have't seen before (as opposed to a new event instance of an issue that's been occurring for a while). These should be top priority to investigate since they represent net-new problems.
```

Comment:
I wish we could include a regressions search too, but there seems to be [outstanding discussion](https://github.com/getsentry/sentry/discussions/43039) around Sentry's limitations around this. I think it'll be OK because users should get emails about regressions; maybe add a note about that?